src/java.desktop/share/classes/javax/swing/border/MatteBorder.java

```diff
@@ -75,6 +75,8 @@ public MatteBorder(int top, int left, int bottom, int right, Color matteColor)
      * Creates a matte border with the specified insets and color.
      * @param borderInsets the insets of the border
      * @param matteColor the color rendered for the border
+     * @throws {@code NullPointerException} if {@code borderInsets}
+     *          is {@code null}
```

Comment:
Would `@inheritDoc` work in this case? The message is the same as in its super class.
```suggestion
     * @throws {@inheritDoc}
```

Reply:
If we updated the parent class should not we do the same for all upto AbstractBorder?

Reply:
I guess we should..updated..